eo-runtime/src/main/java/org/eolang/Param.java

```diff
@@ -78,7 +79,15 @@ public Param(final Phi obj, final String name) {
      */
     public <T> T strong(final Class<T> type) {
         final Object ret = this.weak();
-        if (!type.isInstance(ret)) {
+        final Object result;
```

Comment:
@andreoss I think you are doing this too late. It would be better to change `to-java.xsl` so that this code appears over there. In other words, when we make an instance of the primitive object, we already should make it right, taking the bytes from XMIR, converting to `String` and giving it to the newly created object. Again, this code should be in `to-java.xsl`, not here.